core/connections.go

```diff
@@ -32,7 +32,7 @@ func (c *connections) setSession(region string) {
 
 func (c *connections) connect(region string) {
 
-	logger.Println("Creating Service connections in", region)
+	debug.Println("Creating service connections in", region)
```

Comment:
Before:

```
2020/01/25 17:22:00 main.go:129: Scanning for available AWS regions
2020/01/25 17:22:01 main.go:101: Enabled to run in us-east-1, processing region.
2020/01/25 17:22:01 region.go:78: Creating connections to the required AWS services in us-east-1
2020/01/25 17:22:01 connections.go:35: Creating Service connections in us-east-1
2020/01/25 17:22:01 connections.go:51: Created service connections in us-east-1
2020/01/25 17:22:01 region.go:85: Scanning for enabled AutoScaling groups in  us-east-1
```

The above is repeated per region, so it adds up if running in a lot of regions.


After:
```
020/01/25 17:26:56 main.go:129: Scanning for available AWS regions
2020/01/25 17:26:56 main.go:101: Enabled to run in us-west-2, processing region.
2020/01/25 17:26:56 region.go:78: Creating connections to the required AWS services in us-west-2
2020/01/25 17:26:56 connections.go:35: Creating Service connections in us-west-2
2020/01/25 17:26:56 main.go:101: Enabled to run in sa-east-1, processing region.
2020/01/25 17:26:56 region.go:78: Creating connections to the required AWS services in sa-east-1
2020/01/25 17:26:56 connections.go:35: Creating Service connections in sa-east-1
```

Since we still have the `Enabled to run in us-west-2, processing region` log, I don't think much is missed by removing these.